components/kserve/src/kservedeployer.py

```diff
@@ -421,6 +421,7 @@ def main():
                 del model_status['components']['predictor']['latestRolledoutRevision']
                 del model_status['components']['predictor']['url']
                 del model_status['spec']
+                del model_status["status"]["url"]
```

Comment:
We can add a new parser argument flag to remove the status if this is still exceeding the tekton result limit. 
```
if skip_status:
  model_status = {}
```

Reply:
Okay!